docs/source/standard.md

```diff
@@ -4,27 +4,62 @@
 
 <!-- What this document intends to do (and what it doesn't), the level of detail we aim for contrasted with other technical standards -->
 
-The SATRE standard has been developed based on the following principles:
+## Design
 
-- TREs should be as as easy as possible for end-users to use (_e.g._ researchers) whilst still remaining secure.
-- TRE deployments should be offered that support data of different levels of sensitivity (_e.g._ through a tiered system of technical controls and policies).
-- TREs conforming to the standard should be interoperable and provide a familiar end-user experience.
-- The standard will be managed and updated following an open, community-driven process, and will not be tied to a single vendor or implementation.
+### Overview
+
+The SATRE specification has been designed to follow a capability-evaluation model.
+
+The specification is presented in terms of capabilities teams should aim for across all aspects of TRE provision.
+These capabilities in turn are broken down into components, which are represented by statements of processes, methods and practices teams should have in place to ensure they fulfil the capability requirements.
+
+Any particular component is labeled as `Mandatory`, `Recommended` or `Optional` in order to meet the specification.
+Currently, teams are able to either meet the specification (show they can fulfil all `Mandatory` components) or not.
+Future versions of the specification may introduce more granular levels of evaluation, for instance tiered level of accreditation based on fulfilment of `Mandatory`, `Recommended` and `Optional` components respectively.
```

Comment:
We should have an explanation of what we mean by these terms.

Is it something like,

- Mandatory: We think this is required, if it doesn't have this, it isn't a TRE
- Recommended: We (strongly) feel that TREs should have this. It makes a TRE better
- Optional: We think many TREs would benefit from this but there are reasons to _not_ do this